contracts/utils/uint256_checked_test.cairo

```diff
@@ -0,0 +1,44 @@
+%lang starknet
```

Comment:
not sure if we want test contracts to be in here or `tests/`. otherwise they will be part of the exposed library.